imap_processing/tests/hit/test_decom_hit.py

```diff
@@ -263,6 +240,24 @@ def test_subcom_sectorates(sci_dataset):
         )
 
 
+def test_decompress_rates_16_to_32():
```

Comment:
You could use pytest.mark.parameterize here instead of defining that list of tuples and doing a for loop: https://docs.pytest.org/en/stable/how-to/parametrize.html

Reply:
Thanks for the suggestion! I haven't used this pytest feature yet so it's great to learn a new approach